semgrep/tox.ini

```diff
@@ -11,4 +11,3 @@ setenv =
     PIPENV_VERBOSITY = -1
 
 [pytest]
```

Comment:
Looks like we can remove this whole `[pytest]` key now.

Reply:
```suggestion
```